portfolio/src/main/webapp/script.js

```diff
@@ -84,13 +93,15 @@ function getRandomQuoteUsingArrowFunctions() {
 async function getRandomQuoteUsingAsyncAwait() {
   const response = await fetch('/data');
   const quote = await response.text();
-  document.getElementById('quote-container').innerText = quote;
+  document.getElementById('comment_author').innerText = comment_author;
+  document.getElementById('text-input').innerText = comment;
+  //document.getElementById('quote-container').innerText = quote;
 }
 
 /**
  * Fetches quotess from the servers and adds them to the DOM.
  */
-function getQuote() {
+/*function getQuotesFromServer() {
```

Comment:
I saw that you call "getQuotesFromServer()" in index.html, but this function is commented out? Nonetheless I still think you should make this change in PR for part 3 instead.